ui/pages/confirm-transaction-base/gas-details-item/gas-details-item.test.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+
+import { ETH } from '../../../helpers/constants/common';
+import { renderWithProvider } from '../../../../test/jest';
+import configureStore from '../../../store/store';
+
+import GasDetailsItem from './gas-details-item';
+
+const render = (props) => {
+  const store = configureStore({
+    metamask: {
+      nativeCurrency: ETH,
+      preferences: {
+        useNativeCurrencyAsPrimaryCurrency: true,
+      },
+      provider: {},
+    },
+  });
+
+  return renderWithProvider(<GasDetailsItem txData={{}} {...props} />, store);
+};
+
+describe('GasDetailsItem', () => {
+  it('should should render label', () => {
+    render();
+    expect(screen.queryByText('Gas')).toBeInTheDocument();
```

Comment:
I believe `renderWithProvider` returns `{ queryByText }` as well as all of the other React Testing Library helpers — what do you think about using `const { queryByText } = render()`?

Reply:
This is new recommendation from react-testing-library https://testing-library.com/docs/queries/about/

Reply:
@jpuri I believe Elliot is asking whether we should just get `queryByText` from the object returned `renderWithProvider` call, as done in `ui/components/app/transaction-list-item/transaction-list-item.component.test.js` for example, instead of importing it separately here. Is there any reason that wouldn't work?

Reply:
@jpuri Hmm, I didn't notice that all of the examples in the docs use `screen` instead of the helpers that `render` returns. That's interesting. I think my thought was that the name `screen` might convey that we are finding this globally instead of within the context of a component, in which case we could mistakenly find an element that we don't mean to find. But considering we are always rendering one component at a time then I guess that would never happen, so this isn't a big deal.

Reply:
https://stackoverflow.com/questions/61482418/react-testing-library-screen-vs-render-queries